d8/d8batch/README.md

```diff
@@ -0,0 +1,11 @@
+### D8 Batch
+
+#### tl;dr;
+Drupal's [Batch API](https://api.drupal.org/api/drupal/core!includes!form.inc/group/batch/8) did not change anything in Drupal 8, that is why this demo module also contains other useful examples from the Drupal 8 new features.
```

Comment:
Am I correct that those useful examples are:
- using Guzzle (to fetch something from an URL),
- using Crawler (to walk the XML);
- providing a custom content type?

What happens if a different module already provides the same content type?